### ISSUE TITLE
A new feature to show how much data saved by blokada

### ISSUE THREAD

**Issue by tanmay-bhat** [blokadaorg/blokada#773]
Thanks for reporting issues back to Blokada! Please follow few simple rules:

- **Before** you go ahead and record the bug you found, we would like to ask you to take a look on our [Knowledgebase site](https://support.blokada.org). Maybe it is already solved and the solution is provided there.
- Use search here, on GitHub, to validate if your issue isn't already reported
- Read answers to common questions at http://go.blokada.org/faq
- Having a problem with a particular app? Maybe we know already, check http://go.blokada.org/issue_app
- Blokada not working at all? Maybe your device has known issues, check http://go.blokada.org/issue_device
- Reporting a bug? Your title must answer a question "What does not work for you?" (example: "Blokada randomly closes after few days", but not, "It doesn't work"). Also make sure you follow the rest of the template and provide complete information.
- Requesting a feature? Take a look around [here](https://github.com/blokadaorg/blokada/issues?q=is%3Aissue+is%3Aopen+label%3Atype-F) if that isn't yet requested. Your title must be a full sentence starting with "I want" (example: "I want a widget for turning filtering on/off", but not, "Tasker support"). You can skip the rest of the template.

### Actual behaviour
No broken feature.

### Expected behaviour
Suggestion to include how much data saved in (MB/GB) from using blokada maybe per day/week/month basis on the home page to help/motivate users who rely on mobile data. Also IMO it will help users follow blokada more considering the amount of data it saved. 

### Steps to reproduce  
Let us know how can we get the same result that you have. If it happened only once, please check if you can get the same misbehaviour again. Don't report one-time-only glitches.

- cannot reporduce as ita a suggestion for a feature

### Configuration
1. Blokada version:4.7.3
2. Device model:Redme note 7 pro (violet)
3. Android version / ROM: android 10 MIUI 11

### Questions
1. Do you know about recent updates (for example system update, security patch update)? 

July 2020

2.  How often does this problem happen to you?
NA

**Comment by ps100000** [comment]
Hey. :)
This feature has been requested before sadly it's impossible to tell how large the amount of traffic for a blocked request would have been. 

Duplicate of #416
